features/networking/pod.feature

```diff
@@ -480,3 +480,40 @@ Feature: Pod related networking scenarios
     | ip route |
   Then the output should contain:
     | <%= cb.pod_cidr %> dev <%= cb.tunnel_inf_name %> |
+    
+  # @author anusaxen@redhat.com
+  # @case_id OCP-26373
+  @admin
+  @destructive  
+  Scenario: Pod readiness check for OVN		
+  Given the env is using "OVNKubernetes" networkType
+  And OVN is functional on the cluster
+  #Need this cleanup to restore cluster to healthy state due to future destructive steps
+  And I register clean-up steps:
+  """
+  And OVN is functional on the cluster
+  """
```

Comment:
You can use a system verification step here. They run at the moment as well during clean-up.
```
Given system verification steps are used:
```